aten/src/ATen/native/Normalization.cpp

```diff
@@ -54,6 +54,60 @@ struct Var {
   }
 };
 
+/// A fast path for CPU inference when all tensors are contiguous.
+template<typename scalar_t>
+void batch_norm_cpu_inference_contiguous(Tensor* output, const Tensor& input,
```

Comment:
can you make output "Tensor &" instead?  That's the usual pattern for output parameters currently (you can search in this directory for functions ending in "_out".

Reply:
Done